includes/tasks/class-wc-calypso-task-headstart-products.php

```diff
@@ -46,4 +52,75 @@ public static function has_products() {
 
 		return $product_count > 0;
 	}
+
+	/**
+	 * Title.
+	 *
+	 * @return string
+	 */
+	public function get_title() {
+		if ( self::is_already_selling() ) {
+			return __( 'Import your products', 'woocommerce' );
+		}
+
+		return __( 'Add your products', 'woocommerce' );
+	}
+
+	/**
+	 * Action label.
+	 *
+	 * @return string
+	 */
+	public function get_action_label() {
+		if ( self::is_already_selling() ) {
+			return __( 'Import products', 'woocommerce' );
+		}
+
+		return __( 'Add products', 'woocommerce' );
+	}
+
+	/**
+	 * Content.
+	 *
+	 * @return string
+	 */
+	public function get_content() {
+		if ( self::is_already_selling() ) {
+			return __(
+				'Import your products. Show off your products or services and get ready to start selling – import your existing product info, images, and descriptions.',
+				'woocommerce'
+			);
+		}
+
+		return __(
+			'Add your products. Show off your products or services and get ready to start selling – add your product info, images, and descriptions.',
+			'woocommerce'
+		);
+	}
+
+	/**
+	 * Gets flag for already selling.
+	 *
+	 * @return bool
+	 */
+	public static function is_already_selling() {
+		$data = self::get_onboarding_data();
+		$already_selling_venues = array( 'other', 'brick-mortar', 'other-woocommerce', 'brick-mortar-other' );
```

Comment:
Note that this condition differs compared to the [condition in core](https://github.com/woocommerce/woocommerce/blob/3ae3a0df8dd800d573f330e623fe7b273fa0dbd3/plugins/woocommerce-admin/client/task-lists/fills/utils.js#L15) that determines task fill that gets shown when the task is clicked. The impact is when `selling_venues` is not a value defined in this array will get a "Add product" copy but the task loaded will be "Import" task.

This is because the new behaviour is defined as `If the user skips the step or selects “I’m setting up a store for a client”, default to adding products`. We need to eventually update the condition in core when we work on the project pbIJXs-3mS-p2.